gen3-client/g3cmd/utils.go

```diff
@@ -490,27 +586,30 @@ func initBatchUploadChannels(numParallel int, inputSliceLen int) (int, chan *htt
 }
 
 func batchUpload(furObjects []commonUtils.FileUploadRequestObject, workers int, respCh chan *http.Response, errCh chan error) {
+	// Instantiate interface to Gen3
+	gen3Interface := NewGen3Interface()
```

Comment:
i think for `batchDownload` you passed ref of a `Gen3Interface` into that function to reuse, should we do the same for `batchUpload`?